### DISCUSSION TITLE
Already available resources

### DISCUSSION THREAD

**Discussion by AakashKumarNain** [AakashKumarNain/diffusion_models#1] [Ideas]
# Resources

**Comment by AakashKumarNain** [comment]
These are the resources I found:

1. [**What are Diffusion Models?**](https://lilianweng.github.io/posts/2021-07-11-diffusion-models/) by Lilian Weng
2. [**Score Based Generative Modelling**](https://yang-song.github.io/blog/2021/score/) by  Yang Song
3. [**Introduction to Diffusion Models**](https://www.assemblyai.com/blog/diffusion-models-for-machine-learning-introduction/) Assembly AI
4. [**What are Diffusion Models?**](https://www.youtube.com/watch?v=fbLgFrlTnGU) a video lecture by Ari Seff
5. [**Improving Diffusion Models as an Alternative To GANs**](https://developer.nvidia.com/blog/improving-diffusion-models-as-an-alternative-to-gans-part-1/) by Arash Vahdat and Karsten Kreis
6. [**Annotated Diffusion Model**](https://huggingface.co/blog/annotated-diffusion) by Hugging Face
7. [**Guidance: a cheat code for diffusion models**](https://benanne.github.io/2022/05/26/guidance.html)]


@rish-16 do you have anything to add here before we move on to second discussion?

**Comment by rish-16** [comment]
Hey Aakash, my list has high overlap with yours! 

Maybe, I'll collect some code resources instead of articles.
